integration_tests/sdk/aqueduct_tests/naming_test.py

```diff
@@ -113,22 +113,7 @@ def test_naming_collision_with_different_types(client, data_integration):
     # An overwrite is invalid because the operators are of different types.
     table_artifact = extract(data_integration, DataObject.SENTIMENT, op_name="sql query")
 
-    # Function collides with existing sql artifact
-    _ = extract(data_integration, DataObject.SENTIMENT, op_name="dummy_model")
```

Comment:
Removing this because the following is now allowed:
```
wines = demodb.sql("select * from wine;", name="wine_foo")

@op
def wine_foo(df):
    return df

wines_foo_result = wine_foo(wines)
```